openpgp/packet/config.go

```diff
@@ -48,10 +48,19 @@ type Config struct {
 	// RSABits is the number of bits in new RSA keys made with NewEntity.
 	// If zero, then 2048 bit keys are created.
 	RSABits int
-	// The public key algorithm to use - will always create a signing primary key and encryption subkey.
+	// The public key algorithm to use - will always create a signing primary
+	// key and encryption subkey.
 	Algorithm PublicKeyAlgorithm
 	// Some known primes that are optionally prepopulated by the caller
 	RSAPrimes []*big.Int
+	// AEADConfig configures the Authenticated Encryption with Associated Data.
+	// If a non-nil AEADConfig is passed, use the new AEAD Encrypted Data
+	// Packets defined in the draft of the next version of the OpenPGP
+	// specification.  See the documentation of AEADConfig for more
+	// configuration options related to AEAD.
```

Comment:
Sorry, very last nitpick. Last time I gave a suggestion here I forgot that the comment needed to start with `AEADConfig` 🙈 Right now the text flows kinda strangely. How about something like:

> AEADConfig configures the use of the new AEAD Encrypted Data Packet, defined in the draft of the next version of the OpenPGP specification. If a non-nil AEADConfig is passed, usage of this packet is enabled. By default, it is disabled. See the documentation of AEADConfig for more configuration options related to AEAD.

Reply:
+1, thanks